README.md

```diff
@@ -1,5 +1,548 @@
+      return word[0].toUpperCase() + word.slice(1)
+    }).join(' ')
+  }}
+</div>
+
+
+//good
+<div>{{ normalizedFullName }}</div>
+```
+```javascript
+// The complex expression has been moved to a computed property
+computed: {
+  normalizedFullName() {
+    return this.fullName.split(' ').map(word => {
+      return word[0].toUpperCase() + word.slice(1)
+    }).join(' ');
+  }
+}
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## 7. Quoted attribute values
+<a name="quoted-attribute-values"></a>
+
+> eslint: [`vue/html-quotes`](https://eslint.vuejs.org/rules/html-quotes.html#vue-html-quotes)
+
+**Non-empty HTML attribute values should always be inside quotes (single or double, whichever is not used in JS).**
+
+While attribute values without any spaces are not required to have quotes in HTML, this practice often leads to avoiding spaces, making attribute values less readable.
+
+```html
+//bad
+<input type=text>
+
+<AppSidebar :style={width:sidebarWidth+'px'}>
+
+//good
+<input type="text">
+
+<AppSidebar :style="{ width: sidebarWidth + 'px' }">
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## 7. Directive shorthands
+<a name="directive-shorthands"></a>
+
+**Directive shorthands (`:` for `v-bind:` and `@` for `v-on:`) should always be used.**
+
+```html
+//bad
+<input
+  v-bind:value="newTodoText"
+  v-on:focus="newTodoFocus"
+>
+
+//good
+<input
+  :value="newTodoText"
+  @focus="newTodoFocus"
+>
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## 8. Implicit parent-child communication
+<a name="parent-child-communication"></a>
+
+**Props and events should be preferred for parent-child component communication, instead of this.$parent or mutating props.**
+
+An ideal Vue application is props down, events up. Sticking to this convention makes your components much easier to understand. However, there are edge cases where prop mutation or this.$parent can simplify two components that are already deeply coupled.
+
+The problem is, there are also many simple cases where these patterns may offer convenience. Beware: do not be seduced into trading simplicity (being able to understand the flow of your state) for short-term convenience (writing less code).
+
+[More info](https://vuejs.org/v2/style-guide/#Implicit-parent-child-communication-use-with-caution)
+
+**[⬆ back to top](#table-of-contents)**
+
+## 9. Non-flux state manangement
+<a name="non-flux"></a>
+
+[Vuex](https://github.com/vuejs/vuex) should be preferred for global state management, instead of `this.$root` or a global event bus.
+
+Managing state on `this.$root` and/or using a [global event bus](https://vuejs.org/v2/guide/migration.html#dispatch-and-broadcast-replaced) can be convenient for very simple cases, but are not appropriate for most applications. Vuex offers not only a central place to manage state, but also tools for organizing, tracking, and debugging state changes.
+
+[More info](https://vuejs.org/v2/style-guide/#Non-flux-state-management-use-with-caution)
+
```

Comment:
Could be a good idea having an example on how to access store props from non vue classes?